tcf_website/views/browse.py

```diff
@@ -181,8 +142,8 @@ def course_view(request, mnemonic, course_number):
     # So created a dictionary on the fly to minimize database access
     semesters = {s.id: s for s in Semester.objects.all()}
     for instructor in instructors:
-        instructor.semester_last_taught = semesters.get(
-            instructor.semester_last_taught
+        instructor.semester_last_taught = str(
```

Comment:
I'm not sure, but I swear this code is completely unecessary. This field should be set on the backend as we discussed